spec/factories/ext_management_system.rb

```diff
@@ -76,6 +76,28 @@
           :parent  => :ext_management_system do
   end
 
+  # Traits
+
+  trait :with_clusters do
```

Comment:
can we make the traits local to ems? 
IIUC this makes them available everywhere, but its only useful in :ems, right?